pimd/pim_rp.c

```diff
@@ -124,7 +124,7 @@ void pim_rp_init(struct pim_instance *pim)
 	rp_info->group.family = AF_INET;
 	rp_info->rp.rpf_addr.family = AF_INET;
 	rp_info->rp.rpf_addr.prefixlen = IPV4_MAX_BITLEN;
```

Comment:
above 2 lines can be deleted since `pim_addr_to_prefix` sets it

Reply:
> looks good except 2 lines delete, but I'll spend some time to double check this doesn't break things

This PR will bring below change in behaviour.
When user will configure rp as 255.255.255.255 pim will accept and create not reachable RP.
Earlier it was rejecting this config as bad address.

Reply:
> above 2 lines can be deleted since `pim_addr_to_prefix` sets it

Addressed